spec/system/i18n_js_translations_spec.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+RSpec.describe 'I18n-js Translations' do
+
+  it 'has I18n translations available in javascript', type: :system, js: true do
```

Comment:
`type: :system` is redundant here as it's inferred from the sub-directory of spec/

Reply:
Except Travis seems to need it :( Will leave for now.